### PR TITLE
Bring rfc2629-no-doctype.xslt into lib

### LINKED ISSUE #235: xslt now seems to require rfc2629-no-doctype.xslt to be in lib/
see https://app.circleci.com/pipelines/github/httpwg/http-extensions/465/workflows/8dcfbbb4-29e2-4186-84c5-8ea738c5aeea/jobs/3519

needs to be transformed to 1.0.

### PR DESCRIPTION
**Pull request by mnot**
Fixes #235

### CODE REVIEW

File: main.mk [hunk 1]
```diff
@@ -93,13 +93,16 @@ ifeq (true,$(USE_XSLT))
...
+$(LIBDIR)/clean-for-DTD.xslt: $(LIBDIR)/rfc2629xslt/clean-for-DTD.xslt $(LIBDIR)/rfc2629-no-doctype.xslt
 	$(xsltproc) $(XSLTDIR)/to-1.0-xslt.xslt $< > $@
 
 $(XSLTDIR)/clean-for-DTD.xslt $(XSLTDIR)/rfc2629.xslt: $(XSLTDIR)
 $(XSLTDIR):
 	git clone --depth 10 $(CLONE_ARGS) -b master https://github.com/reschke/xml2rfc $@
 
+$(LIBDIR)/rfc2629-no-doctype.xslt: $(LIBDIR)/rfc2629xslt/rfc2629-no-doctype.xslt
+	$(xsltproc) $(XSLTDIR)/to-1.0-xslt.xslt $< > $@
+
```
Comment (mnot) [inline main.mk:105]: this should probably be moved up.

Comment (martinthomson) [reply main.mk:105]: Yes.

File: main.mk [hunk 2]
```diff
@@ -93,13 +93,16 @@ ifeq (true,$(USE_XSLT))
 $(LIBDIR)/rfc2629.xslt: $(XSLTDIR)/rfc2629.xslt
 	$(xsltproc) $(XSLTDIR)/to-1.0-xslt.xslt $< > $@
 
-$(LIBDIR)/clean-for-DTD.xslt: $(LIBDIR)/rfc2629xslt/clean-for-DTD.xslt
+$(LIBDIR)/clean-for-DTD.xslt: $(LIBDIR)/rfc2629xslt/clean-for-DTD.xslt $(LIBDIR)/rfc2629-no-doctype.xslt
 	$(xsltproc) $(XSLTDIR)/to-1.0-xslt.xslt $< > $@
 
 $(XSLTDIR)/clean-for-DTD.xslt $(XSLTDIR)/rfc2629.xslt: $(XSLTDIR)
```
Comment (mnot) [inline main.mk:99]:
```suggestion
$(LIBDIR)/rfc2629-no-doctype.xslt: $(LIBDIR)/rfc2629xslt/rfc2629-no-doctype.xslt
 	$(xsltproc) $(XSLTDIR)/to-1.0-xslt.xslt $< > $@
 	
$(XSLTDIR)/clean-for-DTD.xslt $(XSLTDIR)/rfc2629.xslt: $(XSLTDIR)
```

File: main.mk [hunk 3]
```diff
@@ -93,13 +93,19 @@ ifeq (true,$(USE_XSLT))
...
+$(LIBDIR)/rfc2629-no-doctype.xslt: $(LIBDIR)/rfc2629xslt/rfc2629-no-doctype.xslt
+ 	$(xsltproc) $(XSLTDIR)/to-1.0-xslt.xslt $< > $@
+ 	
 $(XSLTDIR)/clean-for-DTD.xslt $(XSLTDIR)/rfc2629.xslt: $(XSLTDIR)
 $(XSLTDIR):
 	git clone --depth 10 $(CLONE_ARGS) -b master https://github.com/reschke/xml2rfc $@
 
+$(LIBDIR)/rfc2629-no-doctype.xslt: $(LIBDIR)/rfc2629xslt/rfc2629-no-doctype.xslt
+	$(xsltproc) $(XSLTDIR)/to-1.0-xslt.xslt $< > $@
+
```
Comment (mnot) [inline main.mk:108]:
```suggestion
```

**Comment by mnot** [comment]
ready

**Comment by martinthomson** [comment]
Thanks.  Let me see if I can test this somehow.

**Comment by martinthomson** [comment]
Hmm, I already had a test.  And that test failed because you used spaces instead of tabs.  Yay for makefiles.

**Comment by mnot** [comment]
I used the Web editor. I guess it doesn't like tabs?

**Pull request merged**